### PR TITLE
Testing bigger image with added width & height code

### PR DESCRIPTION
**Pull request by actlatham**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -40,9 +40,11 @@
 	<meta name="twitter:card" content="summary_large_image" />
 	<meta name="twitter:site" content="@getmixim"/>
 	<meta name="twitter:description" content="Mixim is a software tool for remote musical collaboration. Sync and work on your Logic Pro projects with anyone around the world. 10x faster than Dropbox."/>	
-	<meta name="twitter:image" content="https://getmixim.com/images/twitterimagelarge.jpg"/>
+	<meta name="twitter:image" content="https://getmixim.com/images/twitterlargeimage3.jpg"/>
 	<meta name="twitter:title" content="Mixim - the fastest way in the world to make music remotely"/>
 	<meta name="twitter:url" content="https://getmixim.com"/>
+	<meta name="twitter:image:width" content="1144">
+	<meta name="twitter:image:width" content="600">
```
Comment (joerick) [inline index.html]: Should be 'height'?

**Comment by joerick** [comment]
If we're not using the other two twitter images can you remove them?

**Comment by actlatham** [comment]
shit yep - 600 should be height

## 

Alex Latham
alex@getmixim.com

On Wed, Mar 23, 2016, at 10:19 AM, Joe Rickerby wrote:

> If we're not using the other two twitter images can you remove them?
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub[1]

Links:
1. https://github.com/getmixim/getmixim.com/pull/5#issuecomment-200283824

**Comment by actlatham** [comment]
can i just test this first then i will

ᐧ

On Wed, Mar 23, 2016 at 10:19 AM, Joe Rickerby notifications@github.com
wrote:

> If we're not using the other two twitter images can you remove them?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> https://github.com/getmixim/getmixim.com/pull/5#issuecomment-200283824

## 

_Alex Latham_

_Artist Managment_
_Au Mgmt_

_Phone: 07896777345_
_Email: alex@au-mgmt.com alex@au-mgmt.com_
_Follow me on Twitter @actlatham_

**Pull request merged**